libenv/sysinfo.c

```diff
@@ -1352,7 +1352,14 @@ static void OSClasses(EvalContext *ctx)
 #else
 
     char vbuff[CF_MAXVARSIZE];
+
+#ifdef _AIX
+    strlcpy(vbuff, VSYSNAME.version, CF_MAXVARSIZE);
+#endif
+#ifndef _AIX
```

Comment:
You can do `#else` here or if that doesn't work use `#if defined(_AIX)...#else...#endif`.

Reply:
DONE